internal/generator/vector/normalize.go

```diff
@@ -11,7 +11,8 @@ import (
 )
 
 const (
-	FixLogLevel = `
+	ClusterIDFormat = `.openshift.cluster_id = "%s"`
```

Comment:
Can we generate this line based on an env variable? 
```
	ClusterIDFormat = `.openshift.cluster_id = "${OPENSHIFT-CLUSTER-ID:-abc-123-mock-clusterid}"`
```
this way code changes are minimal, dont have to pass around clusterID in the generator functions. Also makes generated conf independent of a specific cluster.

Reply:
Updated to use env var